operator/pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -538,6 +538,7 @@ message GlobalConfig {
 
   // Controls whether one central istiod is enabled.
   google.protobuf.BoolValue centralIstiod = 62;
+
```

Comment:
why change the proto?

Reply:
yeah, this is a leftover from previous interaction. I've noticed it but will only remove it after this one gets approved - trying to make eventual changes in batch.